apps/api/src/domain/models/media-item/entities/raw-media-item-transcript.entity.ts

```diff
@@ -0,0 +1,49 @@
+import { isNonEmptyObject } from '@coscrad/validation-constraints';
+import cloneToPlainObject from '../../../../lib/utilities/cloneToPlainObject';
+import { CoscradDataExample } from '../../../../test-data/utilities';
+import { DTO } from '../../../../types/DTO';
+import BaseDomainModel from '../../base-domain-model.entity';
+
+export enum RawMediaItemTranscriptType {
+    asr = 'Machine Generated (asr)',
+    docx = 'Docx',
+    tsv = 'Track Labels (tsv)',
+}
+
+@CoscradDataExample<RawMediaItemTranscript>({
+    example: {
+        type: RawMediaItemTranscriptType.asr,
+        source: 'whisper-local',
+        version: '25.4',
+        // This needs to be added to views using the event history
+        // timestamp: dummyDateNow,
+        data: null,
+    },
+})
+export class RawMediaItemTranscript extends BaseDomainModel {
```

Comment:
Make a note about how this differs from a standard COSCRAD `Transcript` on an audio or video item.